build.gradle.kts

```diff
@@ -13,6 +13,7 @@ plugins {
 group = "com.gradle"
 
 repositories {
+    mavenLocal()
```

Comment:
This will be removed once the `build-scan-summary` library is published

Reply:
This has been done, though it still requires authentication. Will resolve this comment.